common/gpu/nvidia/prime.nix

```
@@ -3,11 +3,31 @@
 {
   imports = [ ./. ];
 
-  hardware.nvidia.prime = {
-    offload = {
-      enable = lib.mkOverride 990 true;
-      enableOffloadCmd = lib.mkIf config.hardware.nvidia.prime.offload.enable true; # Provides `nvidia-offload` command.
+  options = {
+    hardware.nvidia.primeBatterySaverSpecialisation = lib.mkEnableOption "Configure a specialisation which turns on NVIDIA Prime battery saver";
```

```suggestion
    hardware.nvidia.primeBatterySaverSpecialisation = lib.mkEnableOption "configure a specialisation which turns on NVIDIA Prime battery saver";
```
otherwise the generated option will be named with a capital letter in a random position.